lq1/maps/make.sh

```diff
@@ -130,9 +152,10 @@ function command_help {
     Usage: make.sh [OPERATION] [FLAGS]
 
     OPERATIONS:
-    -h, help      read this msg :3
-    -c, clean     remove extra files that are left after build
-    -m, make      compile all available maps
+    -h, help           read this msg :3
+    -c, clean          remove extra files that are left after build
+    -m, make           compile all available map OR
+    -s, single <MAP>   compile a specified single map
```

Comment:
Please additionally add example usage to the help line, and/or specify in the operation line that the .map extension should not be provided.